mapclassify/classifiers.py

```diff
@@ -1437,11 +1452,13 @@ def _set_bins(self):
             # find an initial solution and then try to find an improvement
             res0 = natural_breaks(x, k)
             fit = res0[2]
-            for i in list(range(self.initial)):
-                res = natural_breaks(x, k)
-                fit_i = res[2]
-                if fit_i < fit:
-                    res0 = res
+            if self.initial > 1:
+                for i in range(self.initial-1):
+                    seed = np.random.random_integers(SEEDRANGE)
+                    res = natural_breaks(x, k, random_state=seed)
```

Comment:
[`natural_breaks` ](https://github.com/sjsrey/mapclassify/blob/b16/mapclassify/classifiers.py#L328) does not have parameter `random_state`. Thus, if `initial`>1, an error will occur.